abilityAuditer.py

```diff
@@ -48,7 +48,7 @@ def getAbilities(url):
 
 	abilities = []
 	expediteCount = defaultdict(int)
-	secondLineStarts = ["This ability", "Cost", "Prereq", "Unlock"]
+	secondLineStarts = ["This ability", "This skill", "Cost", "Prereq", "Unlock"]

```

Comment:
What lines start with "This skill"? I want to make sure that isn't just a typo on my part.

Reply:
https://vennt.fandom.com/wiki/Path_of_the_Detective sometimes uses `This skill` instead of `This ability` and I figured instead of fixing it there, I could just add that line here and make the auditor a little bit more flexible